src/services/OrganizationService.ts

```diff
@@ -300,15 +346,16 @@ const getAllOrganizationsInternalTags = async (): Promise<OrganizationsInternalT
  * Exported functions
  */
 const OrganizationService = {
-  getOrganizations,
-  getOrganizationRoles,
   createOrganization,
-  updateOrganization,
   deleteOrganization,
-  initializeTimeZone,
+  getAllOrganizationsInternalTags,
+  getOrganizationFeatures,
   getOrganizationInternalTags,
+  getOrganizationRoles,
+  getOrganizations,
+  initializeTimeZone,
+  updateOrganization,
   updateOrganizationInternalTags,
-  getAllOrganizationsInternalTags,
```

Comment:
Aside from adding `getOrganizationFeatures`, only change here was to sort exports